### ISSUE TITLE
Workstation Configuration Wizard (simplify key importing)

### ISSUE THREAD

**Issue by deeplow** [freedomofpress/securedrop-workstation#959]
The current way to import the Tails SVS' credentials is a bit manual:
- attaching USB
- decrypting USB
- coping the key to dom0
- copy the journalist interface details to dom0
- in dom0 create a `config.json` with these values

What I am proposing here is a SDW initialization wizard which would be triggered on (first) boot (or when the credentials are not present).
1. Welcome the admin to the SecureDrop Workstation
3. Ask the user to insert the SVS USB (Tails)
4. Ask the user for the decryption key
5. Do the rest in the background
6. Test connection and inform the user in case anything goes wrong

## Open Questions
- including in this flow newsrooms who are installing SecureDrop for the first time and starting only on Workstation (no Tails SVS)
- if this shows on the first boot, then we may want to make sure that the Whonix connection wizard does not get in the way. [Controlling the ISO](https://github.com/freedomofpress/securedrop-workstation/issues/467) and disabling whonix autostart would be away to do this.
- The user should be connected to the internet for the autotesting to work. Should the wizard include some form guidance on connecting to the internet? Or should we leave that to the docs?

**Comment by rocodes** [comment]
I agree with this idea, I think once our provisioning is set up we should definitely have an easier install flow that asks for less information and automates a significant portion of the installation.

I think there are a lot of parts and decisions here that might need to be considered separately (I have put my initial impressions in parentheses but open to discussion): 
- Can we automate a lot more of the install? (Yes)
- Can we do a better job of "validating" the credentials the user provided before install, including network connectivity testing and/or maybe onion service credential testing? (Yes, and there is an existing network check before our updater runs) 
- Can we avoid having the user elaborately construct a config.json or construct any config that requires them to use a text editor? (yes)
    - Can we auto-grab the Submission Private Key (and fingerprint) once the SVS is unlocked (yes)
    - Can we auto-grab the workstation credentials from an admin workstation once the drive is unlocked (yes)   
- Should we take control of asking the user for their drive passphrase(s)? (**No**/maybe/I'm not sure; this is my least favourite part of the export workflow, so I'd say this should be deferred or descoped)
- Should the same VM handle the SVS and the Admin workstation (No; I SVS vm should only be attached to vault, admin VM should be attached to a different VM I think)
- Should the wizard, or a wizard, reappear if valid credentials aren't found? (Yes: relevant https://github.com/freedomofpress/securedrop-workstation/issues/939, leads to below)
- How should the system validate itself at startup and/or watch for or handle config changes (TBD but ongoing/upcoming work is relevant, as are  #936, #951)

This is what I think so far. I'd like

**Comment by rocodes** [comment]
As for the non-Tails SDW, I think that's a good question to keep in mind so we don't box ourselves in with any design choices, but I see it as farther off than a number of our other milestones (blocked on at least #932 , #835 but more broadly https://github.com/freedomofpress/securedrop-client/issues/1725, also https://github.com/freedomofpress/securedrop-client/issues/1104, and probably some others of that ilk)
